src/dotnet-install.ps1

```diff
@@ -957,7 +1060,15 @@ $NormalizedChannel = Get-NormalizedChannel $Channel
 Say-Verbose "Normalized channel: '$NormalizedChannel'"
 $NormalizedProduct = Get-NormalizedProduct $Runtime
 Say-Verbose "Normalized product: '$NormalizedProduct'"
-$DownloadLink = $null
+ValidateFeedCredential
+
+$InstallRoot = Resolve-Installation-Path $InstallDir
+Say-Verbose "InstallRoot: $InstallRoot"
+$ScriptName = $MyInvocation.MyCommand.Name
+($assetName, $dotnetPackageRelativePath) = Resolve-AssetName-And-RelativePath -Runtime $Runtime
+
+$feeds = Get-Feeds-To-Use
+$DownloadLinks = @()
 
 #try to get download location from aka.ms link
 #not applicable when exact version is specified via command or json file
```

Comment:
it would be good to extract this to function too

Reply:
Done. Please see last commits.